dart/lib/src/event_processor/enricher/io_enricher_event_processor.dart

```diff
@@ -63,12 +64,15 @@ class IoEnricherEventProcessor implements EnricherEventProcessor {
       version: _dartVersion,
       rawDescription: Platform.version,
     );
+    final flRuntime = flutterRuntime;
+
     if (runtimes == null) {
-      return [dartRuntime];
+      return [dartRuntime, if (flRuntime != null) flRuntime];
```

Comment:
should we replace the previous `dartRuntime` impl here?

Reply:
Nope, the current one is better but not available everywhere.